versions.json

```diff
@@ -0,0 +1,22 @@
+{
+    "tools": {
+        "kubectl": [
+            "1.31.1",
+            "1.30.5",
+            "1.29.9",
+            "1.28.14"
+        ],
+        "helm": [
+            "3.16.1"
+        ],
+        "powershell": [
+            "7.4.5"
+        ]
+    },
+    "latest": "1.30",
```

Comment:
Is the latest (1.30) supposed to be less than the highest kubectl version available (1.31)?

I'm assuming this is because we are still only officially up to 1.30, as 1.31 is being built but not used

Reply:
yes, that's intentional for now.

The main reason is because the agent it built on the `1.29` SDK, so the drift of +1 should cover `1.30`, but not `1.31` (at least that's the theory).

I just published `1.31` because it's an actual version of k8s :D